XenAdmin/Dialogs/CallHome/CallHomeSettingsDialog.cs

```diff
@@ -149,9 +157,10 @@ private void okButton_Click(object sender, EventArgs e)
                     (int) (frequencyNumericBox.Value * 7),
                     (DayOfWeek) dayOfWeekComboBox.SelectedValue, 
                     (int) timeOfDayComboBox.SelectedValue,
-                    CallHomeSettings.DefaultRetryInterval);
-
-                new SaveCallHomeSettingsAction(pool, newCallHomeSettings, authenticationToken, false).RunAsync();
+                    CallHomeSettings.DefaultRetryInterval);
+
+                new SaveCallHomeSettingsAction(pool, newCallHomeSettings, authenticationToken, usetName, password, false).RunAsync();
+                new TransferCallHomeSettingsAction(pool, newCallHomeSettings, usetName, password, false).RunAsync();
```

Comment:
I think we should use the TransferCallHomeSettingsAction with suppressHistory = true (the last parameter) so it does not show in the Events page.